docs/reference/configuration-reference/alerting-settings.md

```diff
@@ -134,6 +134,18 @@ $$$action-config-email-domain-allowlist$$$
 
     Data type: `string`
 
+`xpack.actions.email.services.ses.host` ![logo cloud](https://doc-icons.s3.us-east-2.amazonaws.com/logo_cloud.svg "Supported on {{ech}}")
+:    The hostname for an Amazon Simple Email Service (SES) service provider that can be used by email connectors.
```

Comment:
```suggestion
:    The SMTP endpoint for an Amazon Simple Email Service (SES) service provider that can be used by email connectors.
```

Monstly to emphasize that HTTPS is not supported and to be more inline with the AWS [terminology](https://docs.aws.amazon.com/general/latest/gr/ses.html).